src/FormField/Calendar.php

```diff
@@ -29,10 +47,21 @@ public function renderView()
             }
         }
 
-        if ($this->type) {
-            $this->options['type'] = $this->type;
+        if (!$this->type) {
+            $this->type = 'datetime';
+        }
+
+        $typeFormat = $this->type.'_format';
+        if ($format = $this->app->ui_persistence->$typeFormat) {
+            $formatter = "function(date, settings){
+                            if (!date) return;
+                            return atk.phpDate('${format}', date);
```

Comment:
can this be passed as an argument into jsExpression?

Reply:
good point